route.go

```diff
@@ -115,6 +128,9 @@ func (r Route) String() string {
 	elems = append(elems, fmt.Sprintf("Flags: %s", r.ListFlags()))
 	elems = append(elems, fmt.Sprintf("Table: %d", r.Table))
 	elems = append(elems, fmt.Sprintf("Realm: %d", r.Realm))
+	if r.Expires != 0 {
+		elems = append(elems, fmt.Sprintf("Expires: %dsec", r.Expires))
```

Comment:
should we put a space b/w before "sec"?

Reply:
This format is inherited from the `ip route` command. Note that the value can be negative because the kernel removes expired entries in a garbage collecting style.